### ISSUE TITLE
os/c-api/il2cpp-config-platforms.h' file not found 

### ISSUE THREAD

**Issue by huzaifahazelGit** [azesmway/react-native-unity#106]
I have spend a lot of time to attempt it  for last one week & tried manys ways to resolve it but remain until falied so kindly guide me properly that how to resolve it .

@
<img width="1512" alt="Screenshot 2024-03-08 at 10 46 29 AM" src="https://github.com/azesmway/react-native-unity/assets/107028141/cc5804e4-a5ae-400a-9c89-67d9dcd72416">

<img width="1512" alt="Screenshot 2024-03-08 at 2 33 17 PM copy" src="https://github.com/azesmway/react-native-unity/assets/107028141/6675696c-7d59-41e8-9228-0b53847097d6">
I have also cleand properly  with this below command
Replace [PRJECT_NAME] with RNUnity
"cd ios; rm -rf ~/Library/Caches/CocoaPods; rm -rf Pods; rm -rf [PROJECT_NAME].xcworkspace; rm -rf ~/Library/Developer/Xcode/DerivedData/*; pod deintegrate; rm -f ios/Podfile.lock; pod setup; arch -x86_64 pod install;”

And also I have proper fallowed the documentation of this below defined package with each & everything step by step in more confident way but still remain error. 
@azesmway/react-native-unity , 

Environment: Unity - 2020.3.42,  Xcode - Version 15.2 (15C500b)

Now Using version

"@azesmway/react-native-unity": "^1.0.7",
   "react": "18.2.0",
   "react-native": “0.72.3”

Also you may look the short video at below for better understanding of this coming issue 

https://drive.google.com/file/d/18LZpOL3uOZZ5Ae5XkGdaQdQGFjrVZrhC/view?usp=sharing

I shall be very thankful to you for your support to resolving

**Comment by Luluno01** [comment]
Thanks for mentioning me but I'm not an expert in dealing with these low-level stuff 🥹, and I haven't been touching anything related to embedding Unity in RN for a while. That said, the error seems to come from the IL2CPP library. To verify if there is a problem with IL2CPP itself, would you please build the Unity project itself as an iOS app?

**Comment by nighthawks** [comment]
@huzaifahazelGit did you succeed to resolve this issue as I have the same problem trying to build for ios.

**Comment by huzaifahazelGit** [comment]
> @huzaifahazelGit did you succeed to resolve this issue as I have the same problem trying to build for ios.
Yes , I have suceed to resolve this issue when I shifted to lower version|
"@azesmway/react-native-unity": "^0.4.0",
